spec/controllers/teams_controller_spec.rb

```diff
@@ -20,4 +20,9 @@
     delete "delete", {id: @team.id}
     expect(@user_with_team.team).not_to eq(nil)
   end
+
```

Comment:
Again, you should check index.html often to see which lines of code are covered or not covered yet.

Reply:
sorry, didn't see your first comment about this